open_q

```diff
@@ -0,0 +1,21 @@
+3. they are parameters for transfor data using http get and we can use them to filtereing or searching specific data
+4. httpresponse for http response and Response used for api respones 
+   httpresponse contain html and Response contain json 
```

Comment:
Response is not limited to json, but can fit multiple types, compared to HttpRespone